templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -360,32 +386,16 @@ Resources:
                   - kms:GenerateDataKey
                   - kms:Decrypt
                 Resource: !Sub arn:${AWS::Partition}:kms:${AWS::Region}:${AWS::AccountId}:alias/aws/sns
-        - PolicyName: !Sub 'securityhub-acct-policy-cfn-${ParentStackName}'
-          PolicyDocument:
-            Version: 2012-10-17
-            Statement:
-              - Effect: Allow
-                Action: cloudformation:ListStackInstances
-                Resource: !Sub arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stackset/AWSControlTowerBP-*
 
         - PolicyName: !Sub 'guardduty-org-policy-${ParentStackName}'
           PolicyDocument:
             Version: 2012-10-17
             Statement:
               - Effect: Allow
                 Action:
-                  - ec2:DescribeInstances
```

Comment:
are these permissions not needed by guardduty ?

Reply:
i checked Access Advisor for Rule, it unused